config/db_config.py

```diff
@@ -0,0 +1,9 @@
+import os
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+POSTGRES_OUT_PORT = os.getenv('POSTGRES_OUT_PORT')
+POSTGRES_USER = os.getenv('POSTGRES_USER')
+POSTGRES_PASSWORD = os.getenv('POSTGRES_PASSWORD')
```

Comment:
where is the name of the database? If you change your RDBMS from Postgres to MySQL - you will not have DB named "postgres" at all.